kubernetes/configs/es1-test/frontend.sh

```diff
@@ -7,6 +7,7 @@ ONETRUST_DOMAIN_SCRIPT_ID="43ec7a2e-aca5-4317-810d-8f80dea732e2"
 SELFCARE_BASE_URL="https://selfcare.pagopa.it"
 API_SIGNAL_HUB_PUSH_INTERFACE_URL="https://raw.githubusercontent.com/pagopa/interop-signalhub-core/refs/heads/main/docs/openAPI/push-signals.yaml"
 API_SIGNAL_HUB_PULL_INTERFACE_URL="https://raw.githubusercontent.com/pagopa/interop-signalhub-core/refs/heads/main/docs/openAPI/pull-signals.yaml"
+SIGNALHUB_PERSONAL_DATA_PROCESS_URL="https://selfcare.uat.interop.pagopa.it/signalhub/pdnd_designazione_responsabilita_dati_personali_sh_v1.pdf"
```

Comment:
since it is not a process
```suggestion
SIGNALHUB_PERSONAL_DATA_DOCUMENT_URL="https://selfcare.uat.interop.pagopa.it/signalhub/pdnd_designazione_responsabilita_dati_personali_sh_v1.pdf"
```